src/shared/api/api.ts

```diff
@@ -92,3 +99,55 @@ export const getAppletUniqueNameApi = ({ name }: AppletUniqueName, signal?: Abor
       signal,
     },
   );
+
+export const postLogFile = (
+  { deviceId, fileId, file }: { deviceId: string; fileId: string; file: FormData },
+  signal?: AbortSignal,
+) =>
+  authApiClient.post<
+    ResponseWithObject<{
+      key: `logfiles/${string}`;
+      url: string;
+      fileId: string;
+    }>
+  >(`/file/log-file/${deviceId}`, file, {
+    params: {
+      fileId,
+    },
+    signal,
+  });
+
+export const postLogFileCheck = (
```

Comment:
do we need postLogFileCheck and getLogFile? 
maybe only leave postLogFile?

Reply:
probably, yes. It's more sutable for mobile app.